vmdb/db/migrate/20150414085027_create_container_routes.rb

```diff
@@ -0,0 +1,21 @@
+class CreateContainerRoutes < ActiveRecord::Migration
+  def up
+    create_table :container_routes do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :creation_timestamp
+      t.string     :resource_version
+      t.string     :namespace
+      t.string     :host
+      t.string     :service_name
+      t.string     :path
```

Comment:
> path is an interesting method name. If you use RelationshipMixin in the future you might get a collision, so we should discuss this column name.

renaming suggestions are welcome. the meaning of this property on openshift side is the relative path on the server that is specified in the host field on which you commented below.
so if host = "www.example.com", path can be /services so later openshift will parse them as www.example.com/services